src/plugins/grid_config/GridConfig.qml

```diff
@@ -286,92 +206,18 @@ GridLayout {
   }
 
   Text {
-    text: "R"
-    color: "dimgrey"
-  }
-
-  IgnSpinBox {
-    Layout.fillWidth: true
-    id: r
-    maximumValue: 1.00
-    minimumValue: 0.00
-    value: 0.7
-    stepSize: 0.01
-    decimals: getDecimals(r.width)
-    onEditingFinished: GridConfig.SetColor(r.value, g.value, b.value, a.value)
-  }
-
-  Text {
-    text: "G"
-    color: "dimgrey"
-  }
-
-  IgnSpinBox {
-    Layout.fillWidth: true
-    id: g
-    maximumValue: 1.00
-    minimumValue: 0.00
-    value: 0.7
-    stepSize: 0.01
-    decimals: getDecimals(g.width)
-    onEditingFinished: GridConfig.SetColor(r.value, g.value, b.value, a.value)
-  }
-
-  Text {
-    text: "B"
-    color: "dimgrey"
-  }
-
-  IgnSpinBox {
-    Layout.fillWidth: true
-    id: b
-    maximumValue: 1.00
-    minimumValue: 0.00
-    value: 0.7
-    stepSize: 0.01
-    decimals: getDecimals(b.width)
-    onEditingFinished: GridConfig.SetColor(r.value, g.value, b.value, a.value)
-  }
-
-  Text {
-    text: "A"
+    Layout.columnSpan: 2
     color: "dimgrey"
+    text: "Grid Color"
   }
 
-  IgnSpinBox {
-    Layout.fillWidth: true
-    id: a
-    maximumValue: 1.00
-    minimumValue: 0.00
-    value: 1.0
-    stepSize: 0.01
-    decimals: getDecimals(a.width)
-    onEditingFinished: GridConfig.SetColor(r.value, g.value, b.value, a.value)
-  }
-
-  Button {
-    Layout.alignment: Qt.AlignHCenter
-    Layout.columnSpan: 4
-    id: color
-    text: qsTr("Custom Color")
-    onClicked: colorDialog.open()
-
-    ColorDialog {
-      id: colorDialog
-      title: "Choose a grid color"
-      visible: false
-      onAccepted: {
-        r.value = colorDialog.color.r
-        g.value = colorDialog.color.g
-        b.value = colorDialog.color.b
-        a.value = colorDialog.color.a
-        GridConfig.SetColor(colorDialog.color.r, colorDialog.color.g, colorDialog.color.b, colorDialog.color.a)
-        colorDialog.close()
-      }
-      onRejected: {
-        colorDialog.close()
-      }
-    }
+  GzColor {
+    id: gzColorGrid
+    Layout.columnSpan: 2
+    Layout.alignment: Qt.AlignRight
+    Layout.bottomMargin: 5
+    Layout.rightMargin: 20
+    onGzColorSet: GridConfig.SetColor(gzColorGrid.r, gzColorGrid.g, gzColorGrid.b, gzColorGrid.a)
   }
 
   // Bottom spacer
```

Comment:
Let's remove this bottom spacer and slightly increase the GzColor's `bottomMargin` to `10`

Reply:
Removing the spacer will cause the empty spaces between the widgets of the plugin to scale:
Without the spacer: 
![bottommargin10](https://user-images.githubusercontent.com/50132891/184226978-953ffd5b-c3cf-4421-a5c4-3ba1985b56ab.png)
With the spacer:
![withSpacer](https://user-images.githubusercontent.com/50132891/184227011-e638e413-aaed-4650-85b2-6b150da5e9c2.png)

Reply:
This only happens because the plugin using the extra room, but if you add another plugin, it will shrink back up. We should remove the spacer because if we add another plugin, then there is this big gap under `Grid Color`.

With spacer:
![image](https://user-images.githubusercontent.com/8602001/184452025-14fe51bc-3d2b-422c-aee6-981cc8273cea.png)


Without spacer:
![image](https://user-images.githubusercontent.com/8602001/184452156-860d3d3c-c1c5-4962-949e-285706ad047b.png)

Reply:
Removed in 89a9ccb